addon/components/universal-export/template.hbs

```diff
@@ -0,0 +1,55 @@
+{{#modal-as-component hidden=hidden}}
+  <div class="modal-header">
```

Comment:
@phndiaye Can you add the .modal-header__borderless class here so that we only have one gray line ? To test this you will need version 0.5.55 of oss :)